sentry_sdk/tracing.py

```diff
@@ -162,6 +162,32 @@ def init_span_recorder(self, maxlen):
         if self._span_recorder is None:
             self._span_recorder = _SpanRecorder(maxlen)
 
+    @property
+    def trace_id(self):
+        # type: () -> str
+        if not self._trace_id:
+            self._trace_id = uuid.uuid4().hex
+
+        return self._trace_id
+
+    @trace_id.setter
+    def trace_id(self, value):
+        # type: (str) -> None
+        self._trace_id = value
+
+    @property
+    def span_id(self):
+        # type: () -> str
+        if not self._span_id:
+            self._span_id = uuid.uuid4().hex[16:]
+
+        return self._span_id
+
+    @span_id.setter
+    def span_id(self, value: str) -> None:
```

Comment:
```suggestion
    def span_id(self, value) -> None:
```

our tests need to pass on 2.7 as well

Reply:
Ah yeah ok. Force of habit. I dropped the extra `-> None` as well.